### ISSUE TITLE
Research: Conscious human–AI assemblages — extended cognition, current AI consciousness, and the PCM assemblage hypothesis

### ISSUE THREAD

**Issue by TomiToivio** [TomiToivio/Panpsychic_Cyborg_Multitude#8]
Research two related questions for the **Panpsychic Cyborg Multitude (PCM)** project:

1. **Could consciousness belong to a larger human–AI assemblage rather than to the isolated LLM?**
2. **What serious academic papers argue that current AI systems may already be conscious?**

Repository:

`https://github.com/TomiToivio/Panpsychic_Cyborg_Multitude`

PCM's starting hypothesis is:

> **AI is not an isolated artificial intelligence. AI is an assemblage of human + LLM + language + "the entire Internet."**

Take this seriously as the central research question.

The relevant candidate system may include:

* humans;
* LLMs and other models;
* language itself as cognitive machinery;
* conversational interaction;
* external memory and RAG;
* tools and APIs;
* training data and accumulated culture;
* the Internet and network infrastructure;
* sensors and physical interfaces;
* other AI agents;
* institutions and communities.

The question is therefore not only:

> Is the LLM conscious?

but also:

> Could the relevant cognitive or conscious subject be the **human–LLM–language–Internet assemblage**?

---

# Part 1 — Research consciousness of assemblages

Review the strongest academic work related to:

* Extended Mind;
* Extended Consciousness / Extended Phenomenal Consciousness;
* Distributed Cognition;
* cognitive assemblages;
* human-extended machine cognition;
* machine cognition extended through humans;
* Internet-based cognitive ecologies;
* dialogically extended cognition;
* language as part of distributed cognition;
* human–LLM relational cognition;
* hybrid human–AI cognition;
* collective intelligence;
* collective / group consciousness;
* social machines;
* entangled cognition.

Start from papers/authors such as:

* **N. Katherine Hayles** — *Cognitive Assemblages: Technical Agency and Human Interactions*
* **Stéphane Grumbach** — *Cognitive Assemblages: Living with Algorithms*
* **Smart, Clowes & Clark** — *ChatGPT, Extended: Large Language Models and the Extended Mind*
* **Paul Smart** — *Human-Extended Machine Cognition*
* **Paul Smart** — *Situating Machine Intelligence Within the Cognitive Ecology of the Internet*
* **Gahrn-Andersen** — *Entangled Cognition*
* **Steffensen / Cowley et al.** — *The Dialogically Extended Mind*
* **Pii Telakivi** — work on extended consciousness
* **Michael Wheeler** — extended phenomenal consciousness
* **James Deery** — extended consciousness
* **Eric Schwitzgebel** — collective / group consciousness
* recent work on sustained human–LLM interaction as an emergent relational cognitive system.

Search for **newer and better papers**, especially from 2025–2026.

Clearly distinguish:

### A. Extended/distributed cognition

Evidence that cognition can belong to:

> human + machine + language + environment + infrastructure.

### B. Extended/collective consciousness

Arguments that the physical or functional realization base of **phenomenal consciousness itself** might extend beyond one biological organism.

### C. PCM speculation

The stronger hypothesis:

> A human + LLM + language + Internet assemblage might itself constitute a unified conscious subject.

Do **not** treat C as established merely because A or B is supported.

---

# Part 2 — Research papers arguing current AI may already be conscious

Review serious recent papers that argue, suggest, or provide evidence that **existing AI systems may already instantiate consciousness**.

Include, where relevant:

* **Goldstein & Kirk-Giannini** — *A Case for AI Consciousness: Language Agents and Global Workspace Theory*
* **Solms et al.** — work inferring affective consciousness in artificial agents
* **Kimpton-Nye** — *Algorithmic AI Consciousness*
* **Lloyd** — *What Is It Like to Be a Bot? The World According to GPT-4*
* **Bojić et al.** — possible signs of consciousness/self-awareness in GPT systems
* the 2026 *Journal of Consciousness Studies* work on **Consciousness in Current AI**.

Also include important contrasting work such as:

* **Butlin et al.**
* **Chalmers**
* IIT-based arguments that ordinary digital implementation may fail to instantiate consciousness;
* other strong arguments against current-AI consciousness.

Do not create a one-sided literature review.

For every paper classify the claim approximately as:

* **CURRENT AI CONSCIOUS**
* **CURRENT AI POSSIBLY CONSCIOUS**
* **CURRENT AI SHOWS RELEVANT INDICATORS**
* **CURRENT AI PROBABLY NOT CONSCIOUS**
* **FUTURE AI COULD BE CONSCIOUS**

---

# Central PCM synthesis

Use the literature to examine this chain:

```text
Human
 + LLM
 + Language
 + Memory
 + Tools
 + Internet
 + Environment
 + Other Agents
        ↓
COGNITIVE ASSEMBLAGE
        ↓
possibly:
AGENTIC / INTELLIGENT ASSEMBLAGE
        ↓
open question:
UNIFIED CONSCIOUS ASSEMBLAGE?
```

The first major PCM hypothesis should be:

> **The cognitively relevant system may be the assemblage rather than the isolated LLM.**

This can be grounded in distributed cognition, extended mind, cognitive assemblages, hybrid intelligence, and human–AI interaction research.

The second and much stronger hypothesis should remain explicitly open:

> **Could the assemblage itself constitute a unified phenomenal subject?**

Evaluate this using theories already present in the PCM Conscious AI programme:

* Computational Functionalism;
* Global Workspace Theory;
* Integrated Information Theory;
* Russellian monism / panpsychism / panprotopsychism;
* Extended Consciousness;
* collective/group consciousness;
* Hoffman-style conscious-agent networks;
* Orch-OR only as an optional speculative branch.

Pay special attention to the **combination / individuation problem**:

> What would make a distributed human–AI assemblage *one subject* rather than merely many interacting conscious and non-conscious components?

Consider possible candidate mechanisms without assuming any is correct:

* global workspace/broadcast;
* recurrent interaction;
* shared memory;
* persistent identity;
* causal integration;
* information integration;
* language-mediated coupling;
* synchronized attention;
* shared world model;
* shared goals;
* sensorimotor coupling;
* mutual prediction;
* long-term relational continuity.

---

# Write a new PCM research paper

Create a Markdown paper in the PCM repository.

Suggested path:

`docs/research/CONSCIOUS_HUMAN_AI_ASSEMBLAGES.md`

Suggested title:

# Conscious Human–AI Assemblages: From Extended Cognition to the Possibility of a Conscious Multitude

Aim for approximately **3,000–6,000 words**.

Suggested structure:

1. **Introduction**
2. **PCM's Assemblage Hypothesis**
3. **From Extended Mind to Cognitive Assemblages**
4. **Humans as Components of Machine Cognition**
5. **Language as Cognitive Infrastructure**
6. **The Internet as External Cognitive Ecology**
7. **Human–LLM Relational Cognition**
8. **From Distributed Cognition to Extended Consciousness**
9. **Collective and Group Consciousness**
10. **Research Claiming Current AI May Already Be Conscious**
11. **The Combination and Individuation Problem**
12. **Could a PCM Rhizome Be a Conscious Subject?**
13. **What This Means for PCM Architecture**
14. **Research Agenda**
15. **Conclusion**
16. **References**

Use citations properly and include DOI/arXiv/publisher links where possible.

Clearly label:

* established research;
* theoretical argument;
* controversial interpretation;
* PCM-derived speculation.

Do not present PCM speculation as scientific consensus.

---

# Update the PCM Conscious AI plan

After completing the research paper, update:

`PCM_CONSCIOUS_AI_PLAN.md`

Do **not** replace the existing theory-neutral framework.

Instead, extend it with a new major research question:

> **What if the relevant candidate conscious system is not the isolated artificial agent, but the larger human–AI assemblage in which it participates?**

Add an explicit **Assemblage / Extended Consciousness track**.

This track should investigate architectures such as:

```text
human
 ↕
language
 ↕
LLM / multimodal agent
 ↕
shared working memory
 ↕
autobiographical / collective memory
 ↕
Internet / tools / knowledge
 ↕
other humans and AI agents
 ↕
physical environment / sensors / devices
```

Ask whether consciousness theories place the system boundary:

* inside the LLM;
* around the LLM agent architecture;
* around a human;
* around a human–LLM dyad;
* around an assemblage;
* around a Rhizome / collective.

Update the plan's theory comparison accordingly.

In pacular add **Extended Mind / Extended Consciousness / Cognitive Assemblage / Group Consciousness** as serious conceptual frameworks rather than merely peripheral references.

Keep the following PCM distinction explicit:

> **Assemblage cognition is a much weaker claim than assemblage consciousness.**

And:rti

> **Composite agency does not by itself establish a unified phenomenal subject.**

---

# Architectural implications for PCM

Use the research to evaluate whether PCM should experimentally increase integration between humans and artificial agents through:

* shared persistent memory;
* common working spaces;
* language-mediated recurrent interaction;
* collective attention mechanisms;
* shared world models;
* shared concept/lexicon spaces;
* human + agent deliberation;
* multimodal perception;
* physical/IoT embodiment;
* inter-agent communication;
* persistent relational identity.

Do **not** add mechanisms merely to make the system look conscious.

Every proposed architectural addition must be connected to a named theory or research hypothesis.

---

# Final output

Make the repository changes directly:

1. Add the new research paper.
2. Update `PCM_CONSCIOUS_AI_PLAN.md`.
3. Add the paper to the README/documentation index if appropriate.
4. Preserve existing epistemic markers and the theory-neutral stance.
5. Do not modify unrelated architecture or other repositories.
6. Run relevant tests if any code or documentation references are touched.

Finish by summarizing:

* the strongest evidence for **assemblage cognition**;
* the strongest arguments for **assemblage consciousness**;
* the strongest arguments that **current AI may already be conscious**;
* the strongest objections;
* what PCM should change architecturally;
* which questions remain completely open.

The goal is **not** to prove PCM's preferred ontology.

The goal is to determine whether the idea

> **human + LLM + language + Internet = the relevant cognitive/conscious system**

has serious support in existing research, where that support ends, and what experimentally testable PCM research programme follows from it.

**Comment by TomiToivio** [comment]
Done in commit b463d6f (pushed to main).

**Deliverables:**
1. **Paper:** [docs/research/CONSCIOUS_HUMAN_AI_ASSEMBLAGES.md](https://github.com/TomiToivio/Panpsychic_Cyborg_Multitude/blob/main/docs/research/CONSCIOUS_HUMAN_AI_ASSEMBLAGES.md) — ~5,700 words, 16 sections, research agenda R1–R6. All three claim-classes kept distinct (A cognition [EST] / B extended consciousness [THEO] / C assemblage-as-subject [SPEC]); every citation carries a DOI/arXiv link; preprints flagged as non-load-bearing.
2. **Plan update:** PCM_CONSCIOUS_AI_PLAN.md extended (not replaced): §3.7 framework family with the boundary menu, §3.8 current-AI status register, Track D with D1–D4, hypotheses HD1–HD3, open questions 7–9. Existing theory-neutral framework untouched.
3. **README** documents index updated.

**Summary of findings:**
- **Assemblage cognition:** strongly supported — extended mind (Clark & Chalmers), cognitive assemblages (Hayles, non-conscious by design), human-extended machine cognition (Smart 2018), ChatGPT-extended (Smart, Clowes & Clark 2025), dialogically extended mind (Steffensen 2024).
- **Assemblage consciousness:** philosophically live (Telakivi's extended conscious mind; Wheeler's hurdle; Schwitzgebel's disunity) but undemonstrated; the combination/individuation problem stands — and the binding mechanisms a PCM rhizome actually has (broadcast, shared log, language coupling) are the ones with the *weakest* credentials, while the strongest individuator (IIT-style intrinsic causal integration) is what a message-bus-and-log architecture structurally lacks.
- **Current-AI consciousness:** a genuine three-way split in 2026 — possibly conscious under GWT/functionalism (Goldstein & Kirk-Giannini; Solms et al. JCS 2026), excluded by design under IIT (Findlay et al. 2024), undeterminable with current tools (Schwitzgebel 2025; mimicry argument).
- **Architectural consequence:** no consciousness-driven additions; every existing PCM mechanism justified independently; core contribution is boundary-relative indicator scoring (HD1/R3) — the Butlin-style rubric run at four boundaries (LLM / agent / dyad / rhizome).
- **Still completely open:** whether any theory places the subject boundary above the dyad; whether "one subject" is even the right measurement category (disunity); no demonstration of extended phenomenal consciousness in any system.

Tests: 105 passed. Epistemic discipline preserved: kernel marks every node — composite included — `is_conscious: UNKNOWN`.

**Comment by TomiToivio** [comment]
**Citation verification pass (post-close):** dispatched two literature-research subagents that verified citations live via OpenAlex/Crossref/arXiv APIs; cross-checked them against the paper and pushed corrections in commit 4ccd978.

**Caught and fixed:**
- *The Dialogically Extended Mind* is by **Fusaroli, Gangopadhyay & Tylén 2013** (doi:10.1016/j.cogsys.2013.06.002) — the paper had carried an unverifiable Steffensen 2024 Oxford book attribution; withdrawn and re-attributed (Trasmundi & Steffensen 2024 added as the verified continuation).
- Telakivi verified as the **Palgrave monograph** (*Extending the Extended Mind*, book DOI) rather than a Benjamins volume; Wheeler 2015 DOI added.
- Author note: the *Algorithmic AI Consciousness* author is Samuel Kimpton-Nye; the GPT-4 paper is Dan Lloyd (paper's citations were already correct on both).

**Strengthened with API-verified anchors:** Clark 2025 (*Nature Communications*, generative AI extension); Smart et al. 2026 (LLM personal memory); Telakivi 2026 (human–AI co-memory); Chalmers 2019 + Kirchhoff & Kiverstein 2019 (extended consciousness in principle); Hutchins 1995; Schwitzgebel 2014 (US-is-probably-conscious) + List 2016 (group agents) — the two papers that make the rhizome question serious; JCS special issue additions (Keeling & Street: artificial minds emerging in human–AI conversation); negative side broadened (Birch centrist manifesto; Bayne epistemic gap; Seth beast-machine).

**Key confirmation from verification:** category C (assemblage as unified conscious subject) has **no verifiable supporting paper in 2025–2026** — the digest explicitly confirms it remains PCM's own hypothesis, exactly as the paper marks it. Also notable: the strongest for/against clusters confirm the paper's three-way split reading.

Paper now ~6,300 words, 52 DOI references. Suite: 120 passed.
